src/acceptance-test/java/org/zalando/nakadi/webservice/EventStreamReadingAT.java

```diff
@@ -416,29 +429,30 @@ private int countStatusCode(final List<CompletableFuture<HttpURLConnection>> fut
         return future;
     }
 
-    @Test(timeout = 10000)
+    @Test(timeout = 60000)
```

Comment:
Just like a previous comment: I think we can replace this with unit test. I don't want to spend up to 60 seconds here.